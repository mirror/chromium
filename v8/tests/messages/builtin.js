function Test() {
  return %StringEquals("a", "b");
}
