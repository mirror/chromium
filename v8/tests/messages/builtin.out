builtin.js:2: Uncaught SyntaxError: Unexpected token %
  return %StringEquals("a", "b");
         ^
